content/departments/product-engineering/engineering/cloud/devops/managed/cost_estimation.md

```diff
@@ -64,4 +65,24 @@ The following summarizes the infrastructure costs incurred for a managed instanc
 | $1.42/GB                       | [Cloud Proxy](https://cloud.google.com/vpc/network-pricing#proxy-instance-charge), 3 are required.                                                                                                                                |
 | $0.12/GB                       | [Internet egress traffic](https://cloud.google.com/vpc/network-pricing#internet_egress) from `us-central1`.                                                                                                                       |
 
+### Cloud SQL usage
+
+Cloud SQL sizing is depending on number of repos
+
+1-500 repos
+
+`db-custom-4-6144`, 100G, $180.26/mo
```

Comment:
in docker-compose, we provision 4Gb for `pgsql` and `code-intel` by default.

I suspect customers would use code-intel from day one, and if they don't, the usage will most likely be zero

Reply:
@rafax any thoughts on CloudSQL pricing?

Reply:
Do you understand how much data we store per 500/1000/2000 repos? How many RAM we need for indices? I'd imagine that we should be able to get away with 2 vCPU instances / 8GB RAM if our dataset is <10GB (and I'd expect <1GB). 

Cloud SQL is ~2x more expensive than underlying compute (per vCPU/RAM), but we should be able to recoup some of that by decreasing MI resource allocation, so this cost sounds reasonable.

Reply:
the data we store in the frontend database is pretty minimal and more so depending on users count over repo count.

Some data point

rv2 has 10K repo and 65 users, pgsql uses 1.6GB disk space

glovo has 1K repo and 250 users, pgsql uses 13GB disk space.

When it comes to code-intel things get tricky. If you don't use LSIF, you will have almost 0 usage, however, if you do, things could get scary depending on your retention policy (how long you keep the index around0. Glovo's codeintel db is alsmot 400Gb.

Reply:
😬 also worth mentioning, `rv2` frontend database was still crashing (OOMKILL) at 50Gb memory (I just bumped it to 70G), we never figure out why.

Reply:
the data we store in the frontend database is pretty minimal and more so depending on users count over repo count.

Some data point

<redacted> customer has 10K repo and 65 users, pgsql uses 1.6GB disk space

<redacted> customer has 1K repo and 250 users, pgsql uses 13GB disk space.

When it comes to code-intel things get tricky. If you don't use LSIF, you will have almost 0 usage, however, if you do, things could get scary depending on your retention policy (how long you keep the index around). Glovo's codeintel db is alsmot 400Gb.